components/mpas-ocean/src/shared/mpas_ocn_tendency.F

```diff
@@ -800,7 +800,19 @@ subroutine ocn_tend_tracer(tendPool, statePool, forcingPool, &
          modifiedConfigName = configBase // '_ttd_forcing'
          call mpas_pool_get_config(ocnConfigs, modifiedConfigName, &
                             config_use_tracerGroup_ttd_forcing)
-
+         if ( groupName == 'freshwaterTracers' ) then
+           if (config_use_tracerGroup_surface_restoring) &
+             call mpas_log_write('surface restoring not supported for freshwaterTracers group')
```

Comment:
I think these (here and below) should be critical errors, not just messages in the log file:
```suggestion
             call mpas_log_write('surface restoring not supported for freshwaterTracers group', messageType=MPAS_LOG_CRIT)
```

Reply:
Good point. I didn't test whether it would still run if the user set these options to true.